src/coprocessor/statistics/analyze.rs

```diff
@@ -108,15 +133,25 @@ impl<S: Snapshot> AnalyzeContext<S> {
 impl<S: Snapshot> RequestHandler for AnalyzeContext<S> {
     async fn handle_request(&mut self) -> Result<Response> {
```

Comment:
We don't have `reschedule().await` here. It will block on yatp.

Reply:
Yes, we should notify `yatp` to reschedule since we fetch rows in batch.